pylot/utils.py

```diff
@@ -1079,3 +1105,54 @@ def run_visualizer_control_loop(control_display_stream):
 def verify_keys_in_dict(required_keys, arg_dict):
     assert set(required_keys).issubset(set(arg_dict.keys())), \
             "one or more of {} not found in {}".format(required_keys, arg_dict)
+
+
+def matches_data(left_msgs, right_msgs):
+    length_matches = (len(left_msgs) == len(right_msgs))
+    return length_matches and all(
+        map(
+            lambda x: x[0].timestamp == x[1].timestamp and x[0].data == x[1].
+            data, zip(left_msgs, right_msgs)))
```

Comment:
Is this for shadow testing and will be removed once the PR is approved?

Reply:
Yeah, this is to check that the messages sent and received are the same.